test/disabled-options.test.js

```diff
@@ -66,3 +66,20 @@ t.test('should not apply the delay header', async t => {
     t.equal(response.headers[HEADERS.delay], undefined)
   })
 })
+
+t.test('should not apply headers if the headers option is false', async t => {
+  const fastify = Fastify()
+  await fastify.register(slowDownPlugin, {
+    headers: false
+  })
+  t.teardown(() => fastify.close())
+
+  fastify.get('/', async () => 'Hello from fastify-slow-down!')
+
+  await slowDownAPI(DEFAULT_OPTIONS.delayAfter, () => fastify.inject('/'))
```

Comment:
I believe there's no need for sending this 1st request.

Reply:
It is needed because I am checking the delay header too. And it exists when the server is starting to delay.
`t.equal(response.headers[HEADERS.delay], undefined)`